katas/content/superposition/three_states_two_qubits_phases/Verification.qs

```diff
@@ -0,0 +1,27 @@
+namespace Kata.Verification {
+    open Microsoft.Quantum.Katas;
+    open Microsoft.Quantum.Math;
+
+    operation ThreeStates_TwoQubits_Phases_Reference (qs : Qubit[]) : Unit is Adj {
+        // First create (|00⟩ + |01⟩ + |10⟩) / sqrt(3) state
+        ThreeStates_TwoQubits_Reference(qs);
+
+        R1(4.0 * PI() / 3.0, qs[0]);
+        R1(2.0 * PI() / 3.0, qs[1]);
+    }
+
+    operation ThreeStates_TwoQubits_Reference (qs : Qubit[]) : Unit is Adj {
+        let theta = ArcSin(1.0 / Sqrt(3.0));
+        Ry(2.0 * theta, qs[0]);
+        ApplyControlledOnInt(0, H, [qs[0]], qs[1]);
+    }
+
+    @EntryPoint()
+    operation CheckSolution() : Bool {
+        return CheckOperationsEquivalenceOnZeroStateWithFeedback(
+            Kata.ThreeStates_TwoQubits_Phases(_),
+            ThreeStates_TwoQubits_Phases_Reference(_),
```

Comment:
A small suggestion, you don't need to use partial application if you're not fixing any of the parameters, you can just pass the operations themselves
```suggestion
            Kata.ThreeStates_TwoQubits_Phases,
            ThreeStates_TwoQubits_Phases_Reference,
```

Reply:
Thank you!